src/site/content/en/blog/websocketstream/index.md

```diff
@@ -154,12 +154,9 @@ which you can then [`write()`](https://developer.mozilla.org/en-US/docs/Web/API/
 data to.
 
 ```js
-  const readWSS = new WebSocketStream(READ_URL);
-  const {readable} = await readWSS.connection;
+  const wss = new WebSocketStream(WSS_URL);
+  const {readable, writable} = await wss.connection;
   const reader = readable.getReader();
-
-  const writeWSS = new WebSocketStream(WRITE_URL);
-  const {writable} = await writeWSS.connection;
   const writer = writable.getWriter();
 
   while (true) {
```

Comment:
Reader and writer connect to different endpoints in the [demo](https://websocketstream-demo.glitch.me/), but I agree that this fact complicates the matter here and your way is better to illustrate usage in general. Thanks.